containing_sevens.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Problem 1
+# This is a one of the quiz problems, so just commit into this repository
+
+# 1. g(1000) = 271
+# 2. computer program to compute g(N)
+
+def g(num)
+  raise ArgumentError, 'Input is not a integer' unless num.is_a?(Integer)
+
+  count = 0
+  (1..num).each do |n|
+    while n > 0
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/NumericPredicate: Use n.positive? instead of n > 0.